tools/publish/processing.js

```diff
@@ -419,8 +419,9 @@ function doCompleteIDL(conf, page, n) {
     utils.forEachNode(doc, function(n) {
       if (n.nodeType == n.ELEMENT_NODE &&
           n.localName == "pre" &&
-          n.getAttribute("class") == "idl") {
```

Comment:
I said I spotted the problem and would send a PR. The problem in https://github.com/w3c/svgwg/pull/495 was in these two lines. `n.getAttribute("class") == "idl"` means that `n.classList.contains("extract")` is always going to be false.

However, I thought this was a plain revert and it is not. This keeps `edit:excludefromidl`, adds "extract" do that IDL block and excludes it from the IDL summary.

It's probably doable to use the extract class in the source as well, but that's an editorial decision that I'm happy to not have an opinion on.

Reply:
Yes, that was an extra point of failure that I noticed (and why I switched it to use a regex test instead), although the main point of failure was still trying to use `classList` and causing the entire script to abort.

Reply:
As for `edit:excludefromidl`, I was mostly concerned that it might be used somewhere else (these build tools are also used by other draft specs), to minimize possible side effects.
(Although I just ran a search & it seems to have been a one-off.) 

But since the end result works for everyone, I'm going to merge & we should get an updated build on svgwg.org within an hour.